.gitlab-ci.yml

```diff
@@ -140,6 +140,7 @@ build-and-lint:
     - yarn build
     - yarn lint
     - node scripts/check-packages.js
+    - node scripts/export-bundles-sizes.js
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏do we plan to export those metrics on all branches or only on main?

Reply:
it's only on main, dashboard will be set to only take metrics when something is merge to main

Reply:
it will be filtered with the branch attribut of the request

Reply:
ok, it is clearer.
Maybe nevermind for now and let's see later with PR reporting what seems to be more convenient between having the branch logic inside the script or in the gitlab configuration 🤷‍♂️